src/core/caniuse.js

```diff
@@ -45,22 +46,26 @@ export async function run(conf, doc, cb) {
   if (!caniuse.feature) {
     return; // no feature to show
   }
-  // Opportunistically insert the style into the head to reduce FOUC.
-  document.head.appendChild(hyperHTML `<style>${caniuseCss}</style>`);
+
+  const link = createResourceHint({
+    hint: "preconnect",
+    href: "https://raw.githubusercontent.com",
+  });
+  document.head.appendChild(link);
+  document.head.appendChild(hyperHTML`<style>${caniuseCss}</style>`);
 
   const parent = document.querySelector(".head dl");
   parent.appendChild(hyperHTML`
     <dt class="caniuse-title" id="${`caniuse-${caniuse.feature}`}">
       Can I Use this API?
     </dt>`);
   const placeholder = parent.appendChild(hyperHTML`
-    <dd class="caniuse-stats">fetching data from caniuse.com...</dd>`);
-
-  const cache = new IDBCache("respec-caniuse", ["caniuse"]);
-  await Promise.all([doc.respecIsReady, cache.ready]);
+    <dd class="caniuse-stats"></dd>`);
```

Comment:
Is there some reason that lines 59-61 can't go in here too? Does hyperHTML get upset for some reason?

Reply:
That'll return two nodes (dd, dt) while `appendChild` accepts a single node (that's why I had earlier wrapped it in a div)